lib/react-super-select.js

```diff
@@ -199,8 +200,13 @@ var ReactSuperSelect = function (_React$Component) {
 
   }, {
     key: 'componentDidUpdate',
-    value: function componentDidUpdate() {
+    value: function componentDidUpdate(prevProps, prevState) {
       this._focusCurrentFocusedId();
+
+      if (this.state.isOpen !== prevState.isOpen) {
+        var openStateCallback = this.state.isOpen ? this.props.onOpenDropdown : this.props.onCloseDropdown;
```

Comment:
Could probably just do this:
`this.state.isOpen ? this.props.onOpenDropdown() : this.props.onCloseDropdown();`
Unless you have eslint set to not allow this :)